read-json.js

```diff
@@ -348,16 +348,9 @@ function mans (file, data, cb) {
     if (er) {
       return cb(er)
     }
-    mans_(file, data, mans, cb)
-  })
-}
-
-function mans_ (file, data, mans, cb) {
-  var m = data.directories && data.directories.man
-  data.man = mans.map(function (mf) {
-    return path.resolve(path.dirname(file), m, mf)
+    data.man = mans
```

Comment:
You probably still want to use path.resolve and then path.relative, to ensure things are normalized

Reply:
lol, fair enough